### PR TITLE
SignIn SignUp Basic Setup

### PR DESCRIPTION
**Pull request by maheshvora**
- Connection with MongoDB
- Developed basic signup with mongo schema
- Developed basic signing with jwt auth token validation
- Added route for get the list tof societys

### CODE REVIEW

File: app/config/auth.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const dotenv = require("dotenv");
+dotenv.config();
```
Comment (ssw1994) [inline app/config/auth.config.js]: I think only one time you need to call dotenv.config? so you can do this in app.js

Comment (ssw1994) [reply app/config/auth.config.js]: also no need to create config file for each variable,we can define all variable in .env file

**Review by avi4js — APPROVED** [review]

File: app/middlewares/authJwt.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                message: "You are not authorized to use our service!!",
+            });
+        }
+        req.userId = decoded.id;
+        next();
+    });
+};
+
+test = (req, res, next) => {
+    console.log("test is run");
```
Comment (ssw1994) [inline app/middlewares/authJwt.js]: can we remove this ,if not needed

**Review by avi4js — APPROVED** [review]

**Review by ssw1994 — APPROVED** [review]

File: app/config/auth.config.js (hunk 1 above)
Comment (maheshvora) [reply app/config/auth.config.js]: Resolved

File: app/middlewares/authJwt.js (hunk 2 above)
Comment (maheshvora) [reply app/middlewares/authJwt.js]: removed

**Pull request merged**